.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
It's not `/node_modules` or `node_modules/` for directory ?

Reply:
if you don't specify a specific folder, this rule will ignore all node_modules folder in the whole project.